drivers/spi/Kconfig.mcux_dspi

```diff
@@ -1,11 +1,43 @@
 # MCUXpresso SDK SPI
```

Comment:
change your commit title to:

drivers: spi: Update mcux driver to support edma

Reply:
ok